riff-raff.yaml

```diff
@@ -0,0 +1,8 @@
+regions: [eu-west-1]
+stacks: [deploy]
```

Comment:
Weird to embed JSON fragments in a yaml file but "I gather" some prefer it this way ;-)